client/stylesheets/_toCat.scss

```diff
@@ -3239,6 +3238,7 @@ On click the images will be displayed at normal size to complete the effect
 .listingItemButtons {
     float: right;
     font-family: 'Roboto';
+    margin-top: 9px !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces